WITNESS ME

Cuboos opened the pull request:
My Wizard Sounds branch wouldn't let me commit any more files to it for some reason, i updated the changelog to include my sound additions.

Cuboos committed:
My Wizard Sounds branch wouldn't let me commit any more files to it for some reason, i updated the changelog to include my sound additions.

Cuboos committed:
My Wizard Sounds branch wouldn't let me commit any more files to it for some reason, i updated the changelog to include my sound additions.
OKAY I DID IT RIGHT THIS TIME, I SWEAR!

Iamgoofball commented on html/changelog.html:
```diff
@@ -59,6 +59,8 @@ <h2 class="date">31 May 2015</h2>
 			<h3 class="author">duncathan updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="tweak">Clarified the ability to retract changeling armblades.</li>
+                <li class="soundadd">Wizard Spells and Wands now have unique sounds, made by Cuboos</li>
```
This needs to be done with the proper changelog system

Cuboos replied:
I fixed it, sorta, i did the right method but now how do i remove the HTML file?

Cuboos committed:
Merge branch 'I'mImportantTooGais' of https://github.com/cuboos/-tg-station into I'mImportantTooGais

Cuboos committed:
Update changelog.html

Screw it, people are talking Coder MumoJumbo, Here's a crude stupid fix that does what i need. I'M LAZY ALRIGHT?

Jordie0608 commented on html/changelogs/Cuboos-I'mImportantTooGais.yml:
```diff
@@ -0,0 +1,5 @@
+delete-after: True
+changes: 
+  -soundadd: "Wizard Spells and Wands/Staffs now have unique sounds"
+  -soundadd: Added Title3.ogg to the start up music
```
You're missing some quotation marks here

Cuboos replied:
Quotations added

On Sun, May 31, 2015 at 10:45 PM, Jordie <notifications@github.com> wrote:

> In html/changelogs/Cuboos-I'mImportantTooGais.yml
> <https://github.com/tgstation/-tg-station/pull/9732#discussion_r31401618>:
>
> > @@ -0,0 +1,5 @@
> > +author: Cuboos
> > +delete-after: True
> > +changes:
> > +  -soundadd: "Wizard Spells and Wands/Staffs now have unique sounds"
> > +  -soundadd: Added Title3.ogg to the start up music
>
> You're missing some quotation marks here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/9732/files#r31401618>.
>

Cuboos committed:
Update Cuboos-I'mImportantTooGais.yml

optimumtact commented:
Consider yourself witnessed, also you fucked up the changelog somehow

I'm gonna guess it's the ' in the changelog yaml file name.

Aranclanos commented:
>more commits than lines changed
how

Cuboos commented:
i... i don't know...

On Mon, Jun 1, 2015 at 7:48 PM, Aranclanos <notifications@github.com> wrote:

> more commits than lines changed
> how
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/9732#issuecomment-107779831>
> .
>

Cheridan commented:
you tried
